primary-key.md

```diff
@@ -4,21 +4,36 @@ summary: The Primary Key constraint specifies that the columns can be used to un
 toc: false
 ---
 
-The Primary Key [constraints](constraints.html) specifies that the columns can be used to uniquely identify rows in a table; this means it combines both the Unique and Not Null constraints.
+The Primary Key [constraint](constraints.html) specifies that the constrained columns' values must uniquely identify each row.
 
-{{site.data.alerts.callout_info}}The Primary Key for a table can only be specified in the <a href="create-table.html"><code>CREATE TABLE</code></a> statement. It can't be changed later using <code>ALTER TABLE</code>.{{site.data.alerts.end}}
+Unlike other constraints which have very specific uses, the Primary Key constraint *should be used for every table* because it provides an intrinsic structure to the table's data. This both makes it easier to understand, as well as improving query performance.
+
+{{site.data.alerts.callout_info}}A table's primary key can only be specified in the <a href="create-table.html"><code>CREATE TABLE</code></a> statement. It can't be changed later using <code>ALTER TABLE</code>, though it is possible to <a href="constraints.html#change-constraints">go through a process</a> to create a new table with the new primary key you want and then migrate the data.{{site.data.alerts.end}}
 
 <div id="toc"></div>
 
 ## Details
 
-- A Primary Key constraint requires that the column values are unique and **do not** contain *NULL* values. You can optionally give the constraint a name using the `CONSTRAINT name` syntax, otherwise the constraint and its associated unique index are called **primary**. 
-   {{site.data.alerts.callout_info}}Strictly speaking, a primary key's unique index is not created, it's derived from the keys under which the data is stored so it takes no additional space. However, it appears as a normal unique index when using commands like <code>SHOW INDEXES</code>.{{site.data.alerts.end}}
-- Columns that are part of a Primary Key are mandatory (NOT NULL). If an optional (nullable) column is made part of a Primary Key, it is made mandatory (NOT NULL). 
+- Tables can only have one primary key.
+- To ensure each row has a unique identifier, the Primary Key constraint combines the properties of both the [Unique](unique.html) and [Not Null](not-null.html) constraints. The properties of both constraints are necessary to make sure each row's primary key columns contain sets of values that do not appear to be the same.
```

Comment:
nit: instead of "values that do not appear to be the same", can't we just say, "values that are not the same"?